src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Components/Header.razor

```diff
@@ -1,18 +1,18 @@
-﻿<header class="header @(true ? "header--authenticated" : null)">
+﻿<header class="header @(false ? "header--authenticated" : null)">
     <div class="header-top-row">
         <div class="header-contact">
-            @if (true)
+            @if (false)
```

Comment:
? @maryamhdr @saeedhalab

Reply:
The purpose of using this `if` condition is to handle the header state if the user is authenticated or not.
We will have a service in the future in which we will handle the user authentication state. Till then, to be able to implement the UI for every state, we have used the very condition.